docs/index.html

```diff
@@ -47,7 +47,8 @@
 <body>
   <div class="links">
     <h1><a href="./float">FLOAT</a></h1>
+    <h1><a href="./cssing/index.html">SLIDE</a></h1>
```

Comment:
Hello, sorry it wasn't clear. Do add a new link with the `<div class="links">` included.
 
For example:
<div class="links">
    <h1><a href="./cssing/index.html">SLIDE</a></h1>
</div>